docs/using_modin.rst

```diff
@@ -62,6 +62,64 @@ Using Modin on a Cluster
 Modin can be run on a cluster, but the setup process is quite complex. We are working on
 a solution to make Modin run on a cluster with a simple setup. More on this coming soon!
 
+Advanced usage (experimental)
+-----------------------------
+
+In some cases, it may be useful to customize your Ray environment. Below, we have listed
+a few ways you can solve common problems in data management with Modin by customizing
+your Ray environment. It is possible to use any of Ray's initialization parameters, all
+of which can be found in `Ray's documentation`_.
+
+.. code-block:: python
+
+   import ray
+   ray.init()
+   import modin.pandas as pd
+
+Modin will automatically connect to the Ray instance that is already running. This way,
+you can customize your Ray environment for use in Modin!
+
+Exceeding memory (Out of core pandas)
+"""""""""""""""""""""""""""""""""""""
+
+If you are working with very large files or would like to exceed your memory, you may
+change the primary location of the DataFrame. If you would like to exceed memory, you
+can use your disk as backup for the memory. This API is experimental in the context of
+Modin. Please let us know what you think!
+
+Here is how you can exceed memory by using your disk:
+
+.. code-block:: python
+
+   import ray
+   num_bytes = 2**40 # Make sure you have disk space to do this!
+   ray.init(plasma_directory="/tmp", object_store_memory=num_bytes)
+   import modin.pandas as pd
+
```

Comment:
Suggestion for next line:
Setting ``plasma_directory="/tmp"`` uses your disk for storing the DataFrame and setting `object_store_memory` sets the maximum size of the plasma store.